lib/resources/parameters.xml

```diff
@@ -427,7 +427,7 @@
       <column name="OBJ_ID" valueNumeric="sys_guid()"/>
       <column name="VER_NBR" value="1"/>
       <column name="PARM_TYP_CD" value="VALID"/>
-      <column name="VAL" value="organization_code=0100;account_delegate_start_date=07/01/2015;account_delegate_principal_name=JMD11;document_type_name=IB"/>
```

Comment:
Can you do something like
<properties>
	<fiscalYear>2016</fiscalYear>
  </properties>

then use it like
<column name="VAL" value="organization_code=0100;account_delegate_start_date=07/01/${fiscalYear};account_delegate_principal_name=JMD11;document_type_name=IB"/>

Just thinking about avoiding duplication